### PR TITLE
Add LA file to class repo

### LINKED ISSUE #18: wvargas10 tracks progress on GitHub Flow
- [x] create a **branch** (git)
- [x] add some commits (git)
- [x] open a *pull request* (GitHub)
- [x] additional commits (git)
- [ ] merge (git)

### PR DESCRIPTION
**Pull request by wvargas10**
This PR adds an LA file to the class repo. This PR closes #18 

- @eabedrapo  do you know any good breweries in LA? :beers: 
- If you were in LA did you visit Hollywood.

### CODE REVIEW

File: wvargas10-LA.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# LA, CA
+
+### Restaurants
+- Where there is good food
```
Comment (githubteacher) [inline wvargas10-LA.md]: @wvargas10 I do remember that LA has fantastic 🍣  -- know of any good place?

File: wvargas10-LA.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+### Attractions for Tourists
+
+- Disney World
+- Universal Studios
+
+### Attractions for Locals
+- Anywhere where people are smiling
```
Comment (rajkorde) [inline wvargas10-LA.md]: in n out :)

**Comment by eabedrapo** [comment]
Hi @wvargas10 

Unfortunately I haven't been in LA and I don't know any attractions there 😭 

If I go there someday, I would def. visit Hollywood, but I'd also like to hike as I've seen that there are some nice places for hiking in LA. @wvargas10 am I wrong?

**Comment by wvargas10** [comment]
I will merge when I have a 1 :+1:  , :ship:  :shipit:

**Comment by eabedrapo** [comment]
👍

**Pull request merged**